Composer/packages/client/src/pages/design/DesignPage.tsx

```diff
@@ -35,37 +51,39 @@ const DesignPage: React.FC<RouteComponentProps<{ dialogId: string; projectId: st
   const activeBot = skillId ?? projectId;
 
   return (
-    <div css={pageRoot}>
-      <Split
-        resetOnDoubleClick
-        initialPrimarySize="20%"
-        minPrimarySize="200px"
-        minSecondarySize="800px"
-        renderSplitter={renderThinSplitter}
-        splitterSize="5px"
-        onMeasuredSizesChanged={onMeasuredSizesChanged}
-      >
-        <SideBar projectId={activeBot} />
-        <div css={contentWrapper} role="main">
-          <CommandBar projectId={activeBot} />
-          <Conversation css={editorContainer}>
-            <div css={editorWrapper}>
-              <Split
-                resetOnDoubleClick
-                initialPrimarySize="65%"
-                minPrimarySize="500px"
-                minSecondarySize="350px"
-                renderSplitter={renderThinSplitter}
-              >
-                <VisualPanel projectId={activeBot} />
-                <PropertyPanel isSkill={activeBot !== projectId} projectId={activeBot} />
-              </Split>
-            </div>
-          </Conversation>
-          <DebugPanel />
-        </div>
-      </Split>
-      <Modals projectId={activeBot} />
+    <div css={root} data-testid={props['data-testid']}>
```

Comment:
why does this need to come from props?

Reply:
It doesn't have to at all. It's a remnant from code copy.